pages/auth/logout.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import Link from "next/link";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { GetServerSideProps } from "next";
+import { useSession } from "next-auth/react";
+
+const Logout = () => {
+  const { i18n, t } = useTranslation("logout");
+  const { data: session } = useSession();
+
+  return (
+    <>
+      <div>
+        <h2 className="gc-h2">{t("messageContent")}</h2>
+        <div className="gc-last-login-time">
+          {t("lastLoginTime")} : {session?.user?.name}
```

Comment:
What are we trying to display here? If I see the screenshot attached to the PR it should be a date but here it looks like you are displaying a username.

Reply:
@craigzour  Please refer to the Unresolved  section. I've addressed this here.